### DISCUSSION TITLE
Log pour debuger

### DISCUSSION THREAD

**Discussion by charlesmerlen** [ssenart/PyGazpar#54] [Q&A]
Bonjour, 

Après installation sur HA, je n'ai aucun résultat. Je ne sais même pas si cela tourne ? Par où commencer ?

**Comment by charlesmerlen** [comment]
Arff, en regardant dans les log de HA, j'obtiens ces logs :

Failed to query PyGazpar library with exception : Traceback (most recent call last): File "/config/custom_components/gazpar/sensor.py", line 142, in update_gazpar_data client.update() File "/usr/local/lib/python3.10/site-packages/pygazpar/client.py", line 58, in update self.__updateLiveMode() File "/usr/local/lib/python3.10/site-packages/pygazpar/client.py", line 118, in __updateLiveMode self.__downloadFile(session, downloadUrl, self.__tmpDirectory) File "/usr/local/lib/python3.10/site-packages/pygazpar/client.py", line 158, in __downloadFile response.raise_for_status() File "/usr/local/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status raise HTTPError(http_error_msg, response=self) requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://monespace.grdf.fr/api/e-conso/pce/consommation/informatives/telecharger?dateDebut=2022-08-30&dateFin=2022-09-29&frequence=Journalier&pceList%5B%5D=85316026896

**Reply by ssenart** [reply]
Hello,

Est-ce que ton numéro de PCE est bon ? Par exemple, le mien contient 14 digits.

Pour faire l'équivalent de ce que fait le programme, tu peux te loguer sur le site GrDF dans "Mon Espace", et ensuite, copier/coller cette URL dans la zone URL pour télécharger le fichier Excel des données.

Je viens de vérifier et cette méthode fonctionne toujours.
